dxr/plugins/python/tests/test_derived.py

```diff
@@ -1,4 +1,5 @@
 from textwrap import dedent
+import nose.tools
```

Comment:
You can go ahead and import more deeply here; that's what we do in most cases, except where names are ambiguous or undescriptive. (`from nose.tools import raises`)